src/util/state-management.spec.js

```diff
@@ -0,0 +1,549 @@
+import assert from 'assert';
+import _ from 'lodash';
+import sinon from 'sinon';
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
If you're using `mount` it means you need the DOM. Make sure you use the `describeWithDOM` helper to make it clear that those tests require the DOM.